pkg/summarizer/summary.go

```diff
@@ -694,18 +694,23 @@ func updateTab(ctx context.Context, tab *configpb.DashboardTab, group *configpb.
 	alert := staleAlert(mod, latest, staleHours(tab))
 	failures := failingTestSummaries(grid.Rows)
 	passingCols, completedCols, passingCells, filledCells, brokenState := gridMetrics(len(grid.Columns), grid.Rows, recent, tab.BrokenColumnThreshold)
+	metrics := tabMetrics(passingCols, completedCols)
+	tabStatus := overallStatus(grid, recent, alert, brokenState, failures)
+	acceptablyFlaky := acceptableFlakiness(passingCols, completedCols, tabStatus, tab.GetStatusCustomizationOptions())
```

Comment:
Ideally, we would add a flag to gate how statuses are calculated. I think you could do that here:
```
acceptablyFlaky := allowFuzzyFlakiness && acceptableFlakiness(...)
```

... then you could pass in the bool in updateTab (and in update, etc...) through the functions, or with a [package-level variable](https://github.com/GoogleCloudPlatform/testgrid/commit/86f7dd7a44839d93611f6cb4506d064e3d4acc48#diff-6f575e683c5fe1eb9301f314392e63c56a00e7b09ae9663adf4c3e4d67f81c70L786-L788).

See [this commit](https://github.com/GoogleCloudPlatform/testgrid/commit/86f7dd7a44839d93611f6cb4506d064e3d4acc48#diff-b2bac5c2bfa8982ae6b2f98b99cd59f6e8722857209295c2bc74baa83c7bc76fL117-L119) for an example (it's deleting a flag, but it shows what I mean by adding the flag)

Reply:
Makes sense, I will add the flag and propagate it all the way down to this function. Thanks!

Reply:
Done @chases2 & @michelle192837 ! I also added a couple of test cases to TestUpdateTab to see the overall behaviours of this newly added feature.